unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/States/EditorMode/FreeCameraMovement.cs

```diff
@@ -191,13 +191,13 @@ private void MouseWheel(float axis)
 
     private void MouseDragRaw(int buttonId, Vector3 mousePosition, float axisX, float axisY)
     {
-        if(buttonId == 1)
+        if(buttonId == 0)
             CameraLook(axisX, axisY);
     }
 
     private void MouseDrag(int buttonId, Vector3 mousePosition, float axisX, float axisY)
     {
-        if (buttonId == 0 ||buttonId == 2)
+        if (buttonId == 2 ||buttonId == 1)
```

Comment:
```suggestion
        if (buttonId == 2 || buttonId == 1)
```
Please be more careful on styling details like this. Remember that this code is going to be read by fellow devs, this is important!